src/applications/vaos/appointment-list/components/CanceledAppointmentsList.jsx

```diff
@@ -116,18 +72,69 @@ function CanceledAppointmentsList({
   }
 
   return (
-    <div className="vads-u-margin-bottom--2 vads-u-background-color--gray-lightest vads-u-padding--2 vads-u-margin-bottom--3">
-      <NoAppointments
-        showScheduleButton={showScheduleButton}
-        isCernerOnlyPatient={isCernerOnlyPatient}
-        startNewAppointmentFlow={() => {
-          recordEvent({
-            event: `${GA_PREFIX}-schedule-appointment-button-clicked`,
-          });
-          startNewAppointmentFlow();
-        }}
-      />
-    </div>
+    <>
+      <div aria-live="assertive" className="sr-only">
+        {hasTypeChanged && 'Showing canceled appointments and requests'}
+      </div>
+      {appointmentsByMonth?.map((monthBucket, monthIndex) => {
+        const monthDate = moment(monthBucket[0].start);
+        return (
+          <React.Fragment key={monthIndex}>
+            <h3
+              id={`appointment_list_${monthDate.format('YYYY-MM')}`}
+              data-cy="canceled-appointment-list-header"
+            >
+              <span className="sr-only">Appointments in </span>
+              {monthDate.format('MMMM YYYY')}
+            </h3>
+            {/* eslint-disable-next-line jsx-a11y/no-redundant-roles */}
+            <ul
+              role="list"
```

Comment:
Since the `ul` elememt already has a default role of `list`, it's not necessary to specify the role here

Reply:
This was recommended by @1Copenut in a previous PR: https://github.com/department-of-veterans-affairs/vets-website/pull/16710#pullrequestreview-631850208

Reply:
Thanks, I'm good with it in that case